pb/protos/profile.proto

```diff
@@ -56,4 +49,20 @@ message Profile {
         string large    = 4;
         string original = 5;
     }
+
+    message Stats {
+        uint32 followerCount  = 1;
+        uint32 followingCount = 2;
+        uint32 listingCount   = 3;
+        uint32 avgRating      = 4;
```

Comment:
Out of curiosity, how does `avgRating` work? Is it the sum of the stars, and it is then divided by `numRatings`?